app/controllers/users_controller.rb

```diff
@@ -7,5 +7,9 @@ def show
     else
       redirect_to new_user_registration_path
     end
+
+    def destroy
+      
+    end
```

Comment:
@naimaseales Review the `UsersController#destroy` action here on lines 11-13. Is it needed?